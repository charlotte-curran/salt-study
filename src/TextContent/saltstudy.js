import { employeeList } from "./employees";
export const saltStudyText = {
  splashPage: {
    header: ["Welcome to", " </salt> ", "!"],
    main: "Pick your location:"
  },
  stockholm: {
    1: {
      main: [
        "learn",
        "how to",
        "code",
        "and",
        "launch",
        "your",
        "career",
        "in tech"
      ],
      button: "for companies!"
    },
    2: {
      header: ["apply", "study for 3 months", "get hired on the spot"],
      main: "Welcome to School of applied technology </salt>",
      footer:
        "Apply to our accelerated career program to become a Full Stack JavaScript developer and get a job within our hiring network of companies. Apply now, we fill our seats on a continuous basis!",
      button: "apply now"
    },
    3: {
      main:
        "’’In our 12 week immersive on-site bootcamp students go from novice to an autonomous Full Stack JavaScript developer working directly with companies in the Epicenter ecosystem.’’",
      video: "https://www.youtube.com/embed/d33cSVnktyA"
    },
    4: {
      header: "Partners and hiring network",
      main: ["epicenter", "campus"],
      main2: "Stockholm & Amsterdam",
      footer:
        "Our students are a part of the ecosystem in Epicenter, interacting with the most innovative companies and using real life cases to accelerate learning. Epicenter is a digital House of Innovation and in a unique partnership </salt> and Epicenter are able to provide the most extensive and targeted hiring network for our coders."
    },
    5: [
      [
        "javascript is the",
        " most sought after ",
        "coding language over the next 3-5 years"
      ],
      [
        "we teach you how to code and how to work",
        " in a real life agile dev team"
      ],
      ["demanding and", " accelerated ", "training and career program"],
      ["europe needs", " 500 000 new technology professionals ", "by 2020!"],
      [
        "our instructors want you to succeed and will be there",
        " every step of the way"
      ],
      [
        "build real products and prototypes for",
        " real tech companies ",
        "during your training"
      ]
    ],
    6: {
      circles: [
        {
          header: "let us launch your career in tech",
          footer:
            "The European tech industry needs you. We are currently missing 500.000 technology professionals in Europe and that number is growing fast."
        },
        {
          header: "we will get you hired by the hottest tech companies",
          footer:
            "We are working with the world’s fastest growing digital companies, and they are looking to hire - offering a salary of 30.000 SEK / 3000 EUR/month."
        },
        {
          header: "complete our career program and the education is free",
          footer:
            "We are investing in you, knowing that you will be a successful software developer consultant within our hiring network."
        }
      ],
      footer: {
        header: "here's how it works:",
        bullets: [
          "Apply and take a test",
          "Get accepted (yaay!)",
          "We sponsor your 3 month full-time coding education at Epicenter",
          "Upon graduation we get you a consultant position within our hiring network with a 30,000 SEK / 3000 EUR per month salary",
          "After doing a kick ass job for 12 months, you get permanently hired by your new company"
        ]
      }
    },
    7: {
      header: "Social Responsibility",
      main: "diversity in tech",
      footer:
        "</salt> is striving for a richer diversity in the European tech industry. We believe that professional organizations should mirror the population's diversity. With our accelerated career program we aim to increase the number women and new Europeans in tech."
    },
    8: {
      header: "this is </salt>",
      slider: employeeList
    },
    9: {
      text: [
        {
          header:
            "we help companies with training and provisioning of software developers",
          footer:
            "Let’s develop a predicable and stable recruitment pipeline of software developers to your company. Enable your team to grow with new hires that have the technical and interpersonal skills they need to succeed. We help decrease your new hires' time to productivity by providing practical, job-specific training, simulating real world agile dev teams for 3 months prior to joining your company."
        },
        {
          header: "hire",
          footer:
            "Bring in diverse and technical talent to your company directly from </salt>. We find, test and recruit the most driven and tech savvy talents to our Career Program were we train your future talents with our industry leading instructors."
        },
        {
          header: "onboard",
          footer:
            "Prepare your new hires with the industry expertise, company knowledge, and skills needed to thrive at your company. We will train your new staff so they are ready to become productive in your dev teams."
        },
        {
          header: "upskill & re-skill",
          footer:
            "Transform your team members through rigorous, intensive and customized training programs in order to keep up with new technologies and business challenges."
        }
      ],
      button: "read more"
    },
    10: {
      header: "join us.",
      address: [
        "Applied Technology STHLM AB",
        "Mäster Samuelsgatan 36",
        "c/o Epicenter Stockholm"
      ],
      addressLink:
        "https://maps.google.com/?q=1200 Mäster+Samuelsgatan+36,+111+57+Stockholm",
      button: "tell me more",
      contact:
        "For general questions please shoot us an e-mail at hello@study-at-salt.com",
      footer: `Wow! We currently experiencing a very high volume of applications and emails, so unfortunately we are not able to reply to all e-mails in a timely manner. We apologize for the inconvenience and we will try to respond as soon as possible.`
    },
    11: [
      "campus epicenter",
      "career program",
      "how does it work?",
      "diversity in tech",
      "our team"
    ],
    12: {
      header: [
        "Send me detailed",
        "information about the",
        "2020 career program"
      ],
      main:
        "Enter your email below to receive our Career Pack with detailed information about the program and how to apply.",
      main2: [
        "Just a heads up, it can take up to ",
        "two hours",
        " to receive your Career Pack!"
      ]
    }
  },
  oslo: {
    1: {
      main: [
        "learn",
        "how to",
        "code",
        "and",
        "launch",
        "your",
        "career",
        "in tech"
      ],
      button: "for companies!"
    },
    2: {
      header: [
        "study code in Stockholm for 3 months",
        "get hired and work in Oslo",
        ""
      ],
      main: "School of applied technology </salt>",
      footer:
        "Apply to our accelerated career program based in Stockholm to become a Full Stack JavaScript developer and get a job within our hiring network of companies in Oslo. We'll pay for accommodation during the training and the required travel. Apply now, we fill our seats on a continuous basis!",
      button: "apply now"
    },
    3: {
      main:
        "’’In our 12 week immersive on-site bootcamp students go from novice to an autonomous Full Stack JavaScript developer working directly with companies in the Epicenter ecosystem.’’",
      video: "https://www.youtube.com/embed/d33cSVnktyA"
    },
    4: {
      header: "Partners and hiring network",
      main: ["epicenter", "campus"],
      main2: "Stockholm & Amsterdam",
      footer:
        "Our students are a part of the ecosystem in Epicenter, interacting with the most innovative companies and using real life cases to accelerate learning. Epicenter is a digital House of Innovation and in a unique partnership </salt> and Epicenter are able to provide the most extensive and targeted hiring network for our coders."
    },
    5: [
      [
        "javascript is the",
        " most sought after ",
        "coding language over the next 3-5 years"
      ],
      [
        "we teach you how to code and how to work",
        " in a real life agile dev team"
      ],
      ["demanding and", " accelerated ", "training and career program"],
      ["europe needs", " 500 000 new technology professionals ", "by 2020!"],
      [
        "our instructors want you to succeed and will be there",
        " every step of the way"
      ],
      [
        "build real products and prototypes for",
        " real tech companies ",
        "during your training"
      ]
    ],
    6: {
      circles: [
        {
          header: "let us launch your career in tech",
          footer:
            "The European tech industry needs you. We are currently missing 500.000 technology professionals in Europe and that number is growing fast."
        },
        {
          header: "we will get you hired by the hottest tech companies",
          footer:
            "We are working with the world’s fastest growing digital companies, and they are looking to hire - offering a salary of 40.000 NOK/month."
        },
        {
          header: "complete our career program and the education is free",
          footer:
            "We are investing in you, knowing that you will be a successful software developer consultant within our hiring network. We will pay for your accommodation in Stockholm during the training."
        }
      ],
      footer: {
        header: "here's how it works:",
        bullets: [
          "Apply and take a test",
          "Get accepted (yaay!)",
          "We sponsor your 3 month full-time coding education at Epicenter in Stockholm",
          "Upon graduation we'll get you a consultant position in Oslo within our hiring network with a 40,000 per month salary",
          "After doing a kick ass job for 12 months, you get permanently hired by your new company"
        ]
      }
    },
    7: {
      header: "Social Responsibility",
      main: "diversity in tech",
      footer:
        "</salt> is striving for a richer diversity in the European tech industry. We believe that professional organizations should mirror the population's diversity. With our accelerated career program we aim to increase the number women and new Europeans in tech."
    },
    8: {
      header: "this is </salt>",
      slider: employeeList
    },
    9: {
      text: [
        {
          header:
            "we help companies with training and provisioning of software developers",
          footer:
            "Let’s develop a predicable and stable recruitment pipeline of software developers to your company. Enable your team to grow with new hires that have the technical and interpersonal skills they need to succeed. We help decrease your new hires' time to productivity by providing practical, job-specific training, simulating real world agile dev teams for 3 months prior to joining your company."
        },
        {
          header: "hire",
          footer:
            "Bring in diverse and technical talent to your company directly from </salt>. We find, test and recruit the most driven and tech savvy talents to our Career Program were we train your future talents with our industry leading instructors."
        },
        {
          header: "onboard",
          footer:
            "Prepare your new hires with the industry expertise, company knowledge, and skills needed to thrive at your company. We will train your new staff so they are ready to become productive in your dev teams."
        },
        {
          header: "upskill & re-skill",
          footer:
            "Transform your team members through rigorous, intensive and customized training programs in order to keep up with new technologies and business challenges."
        }
      ],
      button: "read more"
    },
    10: {
      header: "join us.",
      address: [
        "Applied Technology STHLM AB",
        "Mäster Samuelsgatan 36",
        "c/o Epicenter Stockholm"
      ],
      addressLink:
        "https://maps.google.com/?q=1200 Mäster+Samuelsgatan+36,+111+57+Stockholm",
      button: "tell me more",
      contact:
        "For general questions please shoot us an e-mail at hello@study-at-salt.com",
      footer: `Wow! We currently experiencing a very high volume of applications and emails, so unfortunately we are not able to reply to all e-mails in a timely manner. We apologize for the inconvenience and we will try to respond as soon as possible.`
    },
    11: [
      "campus epicenter",
      "career program",
      "how does it work?",
      "diversity in tech",
      "our team"
    ],
    12: {
      header: [
        "Send me detailed",
        "information about the",
        "2020 career program"
      ],
      main:
        "Enter your email below to receive our Career Pack with detailed information about the program and how to apply.",
      main2: [
        "Just a heads up, it can take up to ",
        "two hours",
        " to receive your Career Pack!"
      ]
    }
  }
};
