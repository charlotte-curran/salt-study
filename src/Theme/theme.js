export const SaltTheme = {
  color: {
    dark: "#222324",
    blue: "#92b3d2",
    white: "rgba(255,255,255, 1)",
    orange: "#ff7a61",
    shadow: "#545353",
    address: "rgba(255, 255, 255, 0.4)",
    navbar: "rgba(34, 35, 36, 0.7)"
  },
  fonts: {
    futuraPt: '"futura-pt", sans-serif',
    raleway: '"raleway", sans-serif'
  }
};
