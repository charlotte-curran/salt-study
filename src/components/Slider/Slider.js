import React, { useState, useRef } from "react";
import { Swipeable } from "react-swipeable";
import { useSpring, config } from "react-spring";
import {
  SliderUI,
  InnerWrapper,
  Employee,
  Headshot,
  EmployeeName,
  Title,
  Text,
  ButtonWrapperInner,
  ButtonWrapperOuter,
  Button,
  Arrow
} from "./Slider.styles";
import { useWindowDimensionsContext } from "../../Context/dimensionsContext";

const Slider = props => {
  const [page, setPage] = useState(0);
  const wrapperRef = useRef(null);
  const { content } = props;

  const { width, height } = useWindowDimensionsContext();

  const getEmployeePadding = () => {
    if (wrapperRef.current) {
      const wrapperWidth = wrapperRef.current.offsetWidth;
      return wrapperWidth / 8;
    } else if (width < 1000 || (width === 1024 && height === 1366)) {
      return width / 8;
    } else if (width >= 1000) {
      return (width * 0.5734) / 8;
    }
  };

  const buttonCounterArray =
    content.length % 2 === 0
      ? new Array(content.length / 2).fill(undefined)
      : new Array((content.length + 1) / 2).fill(undefined);

  const sliderProps = useSpring({
    transform: `translate3d(${
      wrapperRef.current ? -wrapperRef.current.offsetWidth * page : 0
    }px, 0, 0)`,
    config: width > 599 ? config.gentle : config.default
  });

  return (
    <SliderUI ref={wrapperRef}>
      <Swipeable
        onSwipedLeft={() => {
          setPage(page < buttonCounterArray.length - 1 ? page + 1 : 0);
        }}
        onSwipedRight={() => {
          setPage(page > 0 ? page - 1 : buttonCounterArray.length - 1);
        }}
        stopPropagation
        delta={5}
        flickThreshold={0.3}
      >
        <Arrow
          xmlns="http://www.w3.org/2000/svg"
          pointLeft={true}
          onClick={() => {
            setPage(page > 0 ? page - 1 : buttonCounterArray.length - 1);
          }}
        >
          <path d="M9 18l6-6-6-6" />
        </Arrow>

        <Arrow
          xmlns="http://www.w3.org/2000/svg"
          right={width}
          onClick={() => {
            setPage(page < buttonCounterArray.length - 1 ? page + 1 : 0);
          }}
        >
          <path d="M9 18l6-6-6-6" />
        </Arrow>

        <InnerWrapper
          style={sliderProps}
          employeeMargin={getEmployeePadding()}
          width={width}
        >
          {content.map((value, i) => {
            return (
              <Employee
                key={`Employee${i}`}
                margin={getEmployeePadding()}
                width={width}
                isEven={i % 2 === 0 ? true : false}
                id={`employee${i}`}
              >
                <Headshot
                  background={require(`../../images/${value.img}`)}
                  key={`Headshot${i}`}
                />
                <EmployeeName key={`EmployeeName${i}`}>
                  {value.employee_name}
                </EmployeeName>
                <Title key={`Title${i}`}>{value.title}</Title>
                <Text width={getEmployeePadding()} key={`Text${i}`}>
                  {value.footer}
                </Text>
              </Employee>
            );
          })}
        </InnerWrapper>

        <ButtonWrapperOuter>
          <ButtonWrapperInner>
            {buttonCounterArray.map((value, i) => {
              return (
                <Button
                  key={`Button${i}`}
                  isBlue={page === i ? true : false}
                  onClick={
                    page !== i
                      ? () => {
                          setPage(i);
                        }
                      : null
                  }
                />
              );
            })}
          </ButtonWrapperInner>
        </ButtonWrapperOuter>
      </Swipeable>
    </SliderUI>
  );
};

export default Slider;
