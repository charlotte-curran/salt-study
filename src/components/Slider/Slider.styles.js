import styled from "styled-components";
import { animated } from "react-spring";

export const SliderUI = styled.div`
  width: 100%;
  overflow-x: hidden;
  overflow-y: auto;
  height: 86vh;

  @media (max-height: 620px) {
    height: auto;
  }

  /* iphone 6/7/8 plus*/
  @media (width: 414px) and (height: 736px) {
    height: 81vh;
  }

  /* iphone X */
  @media (width: 375px) and (height: 812px) {
    height: 77vh;
  }

  /* galaxy s8 */
  @media (width: 360px) and (height: 740px) {
    height: 80vh;
  }

  /* ipad portrait */
  @media (width: 768px) and (height: 1024px) {
    height: 65vh;
  }

  /* ipad pro landscape */
  @media (width: 1366px) and (height: 1024px) {
    height: 66vh;
  }

  /* ipad pro portrait*/
  @media (width: 1024px) and (height: 1366px) {
    height: 45vh;
  }
`;

export const Employee = styled.div`
  color: ${({ theme }) => theme.color.white};
  font-family: ${({ theme }) => theme.fonts.futuraPt};
  text-align: center;
  margin-left: ${({ width, isEven }) => (isEven ? `${width * 0.1}px` : "0px")};
  margin-right: ${({ width, isEven }) =>
    isEven
      ? `${width - (width / 2 > 250 ? width / 2 : 250) - width * 0.1}px`
      : `0px`};
  max-height: 500px;

  @media (min-width: 600px) {
    margin-left: ${({ margin }) => `${margin}px`};
    margin-right: ${({ margin }) => `${margin}px`};
  }
`;

export const Headshot = styled.div`
  background: ${({ background }) => `url(${background})`};
  background-size: cover;
  background-repeat: no-repeat;
  background-position: center center;
  border-radius: 50%;
  width: 120px;
  height: 120px;
  margin: 0 auto 10% auto;
  filter: grayscale(50%);
`;

export const EmployeeName = styled.div`
  font-size: 17px;
  margin-bottom: 5px;
`;

export const Title = styled.div`
  font-style: italic;
  font-size: 13px;
  margin-bottom: 10%;

  @media (min-width: 1000px) {
    font-size: 14px;
  }
`;

export const Text = styled.div`
  font-size: 13px;
  width: ${({ width }) => (width * 2 >= 125 ? `${width * 2}px` : "125px")};

  @media (min-width: 1000px) {
    font-size: 14px;
  }
`;

export const ButtonWrapperInner = styled.div`
  width: 150px;
  display: flex;
  justify-content: space-between;
`;

export const ButtonWrapperOuter = styled.div`
  display: flex;
  justify-content: center;
  margin-bottom: 2vh;
  margin-top: 2vh;

  @media (max-height: 620px) {
    margin-top: 10px;
  }
`;

export const Button = styled.button`
  background: ${({ isBlue, theme }) =>
    isBlue ? theme.color.blue : theme.color.white};
  width: 6px;
  height: 6px;
  padding: 0;

  @media (min-width: 1000px) {
    width: 6px;
  }
`;

export const Arrow = styled.svg`
  position: relative;
  top: 30vh;
  left: ${({ pointLeft, right }) => (pointLeft ? "2vw" : `${right * 0.911}px`)};
  z-index: 5;
  width: 18px;
  height: 18px;
  stroke: #fff;
  stroke-width: 2;
  stroke-linecap: square;
  transform: ${({ pointLeft }) => (pointLeft ? "rotate(180deg)" : null)};
  cursor: pointer;

  :hover {
    stroke: ${({ theme }) => theme.color.blue};
  }

  @media (min-width: 1000px) {
    left: ${({ pointLeft, right }) =>
      pointLeft ? "2vw" : `${right * 0.542}px`};
  }

  @media (max-width: 600px) {
    display: none;
  }

  /* ipad pro portrait*/
  @media (width: 1024px) and (height: 1366px) {
    top: 15vh;
    left: ${({ pointLeft, right }) =>
      pointLeft ? "2vw" : `${right * 0.94}px`};
  }
`;

export const InnerWrapper = animated(styled.div`
  display: flex;
  height: 100%;
  margin-bottom: 4vh;
  #employee0 {
    margin-left: 5%;
  }

  @media (min-width: 600px) {
    #employee0 {
      margin-left: ${({ employeeMargin }) => `${employeeMargin}px`};
    }
  }

  @media (min-width: 1000px) {
    margin-bottom: 4.6%;
  }

  @media (max-height: 620px) {
    margin-bottom: 0;
  }

  /* ipad pro portrait*/
  @media (width: 1024px) and (height: 1366px) {
    height: 36vh;
  }
`);
