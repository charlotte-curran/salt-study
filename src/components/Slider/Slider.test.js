import React from "react";
import { create } from "react-test-renderer";
import Slider from "./Slider";
import { ThemeProvider } from "styled-components";
import { SaltTheme } from "../../Theme/theme";
import { saltStudyText } from "../../TextContent/saltstudy";
import WindowDimensionsProvider from "../../Context/dimensionsContext";

describe("Slider", () => {
  test("matches the snapshot", () => {
    const slider = create(
      <WindowDimensionsProvider>
        <ThemeProvider theme={SaltTheme}>
          <Slider content={saltStudyText[8].slider} />
        </ThemeProvider>
      </WindowDimensionsProvider>
    );
    expect(slider.toJSON()).toMatchSnapshot();
  });
});
