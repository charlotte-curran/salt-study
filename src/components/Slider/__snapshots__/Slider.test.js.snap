// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider matches the snapshot 1`] = `
<div
  className="sc-bdVaJa bcYptp"
>
  <div>
    <svg
      className="sc-dnqmqq dHwVuP"
      onClick={[Function]}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M9 18l6-6-6-6"
      />
    </svg>
    <svg
      className="sc-dnqmqq dqdrcK"
      onClick={[Function]}
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M9 18l6-6-6-6"
      />
    </svg>
    <div
      className="sc-iwsKbI iRLBXc"
      style={
        Object {
          "transform": "translate3d(0px, 0, 0)",
        }
      }
      width={1024}
    >
      <div
        className="sc-bwzfXH dIYVNa"
        id="employee0"
        width={1024}
      >
        <img
          className="sc-htpNat dBSrOR"
          src="koshi_hamedi.png"
        />
        <div
          className="sc-bxivhb fvQhja"
        >
          Koshi Hamedi
        </div>
        <div
          className="sc-ifAKCX pPXDO"
        >
          Director of strategy at Microsoft, Co-founder at &lt;/salt&gt;
        </div>
        <div
          className="sc-EHOje jRprex"
          width={73.3952}
        >
          Founder of Donya Labs and Director of strategy at Microsoft.
                Koshi came to Sweden in the 80's as a refugee. Through his entrepreneurial drive and determination,
                he built one of Sweden's fastest growing technology companies and established the
                company as the global market leader. Donya Labs was acquired by Microsoft in 2017.
        </div>
      </div>
      <div
        className="sc-bwzfXH boKrr"
        id="employee1"
        width={1024}
      >
        <img
          className="sc-htpNat dBSrOR"
          src="nicole_henning.png"
        />
        <div
          className="sc-bxivhb fvQhja"
        >
          Nicole Henning
        </div>
        <div
          className="sc-ifAKCX pPXDO"
        >
          COO at &lt;/salt&gt;
        </div>
        <div
          className="sc-EHOje jRprex"
          width={73.3952}
        >
          Nicole is a people person with a passion for business and developing others!
                She has worked as a consultant within communication and leadership at several of Sweden's largest
                companies within tech and telecom. Most recently she comes from the recruitment industry.
                Core values for Nicole are teamwork, trust, ethics, and that there's always a reason to celebrate!
        </div>
      </div>
      <div
        className="sc-bwzfXH dIYVNa"
        id="employee2"
        width={1024}
      >
        <img
          className="sc-htpNat dBSrOR"
          src="marcus_hammarberg.jpg"
        />
        <div
          className="sc-bxivhb fvQhja"
        >
          Marcus Hammarberg
        </div>
        <div
          className="sc-ifAKCX pPXDO"
        >
          Head of curriculum & agile coach at &lt;/salt&gt;
        </div>
        <div
          className="sc-EHOje jRprex"
          width={73.3952}
        >
          In parallel with &lt;/salt&gt; Marcus is an agile coach at Spotify. As a developer, Marcus contributes
               to open source frameworks and he has also produced several courses on JavaScript development for PluralSight,
               Marcus has also co-authored the book "Kanban In Action". Marcus has worked at companies such as Tradera,
               Huawei and Spotify
        </div>
      </div>
      <div
        className="sc-bwzfXH boKrr"
        id="employee3"
        width={1024}
      >
        <img
          className="sc-htpNat dBSrOR"
          src="ersan_curuklu.png"
        />
        <div
          className="sc-bxivhb fvQhja"
        >
          Erşan Curuklu
        </div>
        <div
          className="sc-ifAKCX pPXDO"
        >
          UX instructor at &lt;/salt&gt;
        </div>
        <div
          className="sc-EHOje jRprex"
          width={73.3952}
        >
          Erşan is a highly experienced requirements analyst and UX Designer with over 15 years of experience
                as a digital designer. Erşan is a teacher and lecturere in the UX area, for e.g. at Bergh's School of Communications
                in Stockholm. Erşan has won several awards in UX/UI, including Svenska Design Priset 2014 and Resumé 2014.
        </div>
      </div>
      <div
        className="sc-bwzfXH dIYVNa"
        id="employee4"
        width={1024}
      >
        <img
          className="sc-htpNat dBSrOR"
          src="max_hamedi.png"
        />
        <div
          className="sc-bxivhb fvQhja"
        >
          Max Hamedi
        </div>
        <div
          className="sc-ifAKCX pPXDO"
        >
          Professor at KTH & Harvard, Co-founder at &lt;/salt
        </div>
        <div
          className="sc-EHOje jRprex"
          width={73.3952}
        >
          Postdoctorial research fellow at Harvard University & Associate Professor at the Royal Institute of Technology in
                Stockholm. Our in-house doctor and professor scanning for future technologies, driving our efforts to provide the best applicable
                technology training possible.
        </div>
      </div>
      <div
        className="sc-bwzfXH boKrr"
        id="employee5"
        width={1024}
      >
        <img
          className="sc-htpNat dBSrOR"
          src="richard_andemark.jpg"
        />
        <div
          className="sc-bxivhb fvQhja"
        >
          Richard Andemark
        </div>
        <div
          className="sc-ifAKCX pPXDO"
        >
          CEO at &lt;/salt&gt;
        </div>
        <div
          className="sc-EHOje jRprex"
          width={73.3952}
        >
          At age 25 Richard launched a global SaaS company's operations in Finland as Managing Director, reaching profitability
                in just six months. At 29, Richard launched Donya Labs' American operations in San Francisco as GM, achieving exponential
                revenue growth and the company was acquired by Microsoft.
        </div>
      </div>
      <div
        className="sc-bwzfXH dIYVNa"
        id="employee6"
        width={1024}
      >
        <img
          className="sc-htpNat dBSrOR"
          src="daniel_nasman.jpg"
        />
        <div
          className="sc-bxivhb fvQhja"
        >
          Daniel Näsman
        </div>
        <div
          className="sc-ifAKCX pPXDO"
        >
          Head instructor at &lt;/salt&gt;
        </div>
        <div
          className="sc-EHOje jRprex"
          width={73.3952}
        >
          Daniel most recently comes from Daniel Wellington where he was Global Head of IT Development, overseeing development offices in Sweden, Poland and China. 
                With many years experience from being both a developer and manager in an international setting he knows how important the developer is on todays and 
                tomorrows market. He is passionate about building teams and making people grow. 
        </div>
      </div>
      <div
        className="sc-bwzfXH boKrr"
        id="employee7"
        width={1024}
      >
        <img
          className="sc-htpNat dBSrOR"
          src="mies_mandoki_cirillo.jpg"
        />
        <div
          className="sc-bxivhb fvQhja"
        >
          Mies Mandoki Cirillo
        </div>
        <div
          className="sc-ifAKCX pPXDO"
        >
          Instructor at &lt;/salt&gt;
        </div>
        <div
          className="sc-EHOje jRprex"
          width={73.3952}
        >
          Mies graduated from the first class of &lt;/salt&gt; developers. He has a huge passion for the digital space and is always eager to
                learn new tech. Before &lt;/salt&gt;, he was working with video editing, graphic design and was the main producer of a large ongoing video series.
                Mies is a calm, highly driven individual who firmly believes that coders are the superstars of the future.
        </div>
      </div>
      <div
        className="sc-bwzfXH dIYVNa"
        id="employee8"
        width={1024}
      >
        <img
          className="sc-htpNat dBSrOR"
          src="zachary_eriksson.jpg"
        />
        <div
          className="sc-bxivhb fvQhja"
        >
          Zachary Eriksson
        </div>
        <div
          className="sc-ifAKCX pPXDO"
        >
          Junior Instructor at &lt;/salt&gt;
        </div>
        <div
          className="sc-EHOje jRprex"
          width={73.3952}
        >
          Zachary graduated from the third class of developers. His passion for technology and computers has always been close to his heart.
                Before &lt;/salt&gt; he worked as a consultant within Wordpress development and design. Zachary is an energetic, social and driven individual who
                believes communication is the key to success.
        </div>
      </div>
      <div
        className="sc-bwzfXH boKrr"
        id="employee9"
        width={1024}
      >
        <img
          className="sc-htpNat dBSrOR"
          src="stina_johansson.jpg"
        />
        <div
          className="sc-bxivhb fvQhja"
        >
          Stina Johansson
        </div>
        <div
          className="sc-ifAKCX pPXDO"
        >
          Head of People & Talent at &lt;/salt&gt;
        </div>
        <div
          className="sc-EHOje jRprex"
          width={73.3952}
        >
          Stina has a passion for people and developing new talents, employees and oganisations! Before &lt;/salt&gt; Stina worked as a consultant, project
                manager and with human resources. She has a long experience from the recruitment-, bank-, and IT industries. Core values for Stina is curiosity,
                commitment, courage, professionalism and last but not least to have fun along the way.
        </div>
      </div>
      <div
        className="sc-bwzfXH dIYVNa"
        id="employee10"
        width={1024}
      >
        <img
          className="sc-htpNat dBSrOR"
          src="sara_hjortmar.jpg"
        />
        <div
          className="sc-bxivhb fvQhja"
        >
          Sara Hjortmar
        </div>
        <div
          className="sc-ifAKCX pPXDO"
        >
          Sales & Marketing &lt;/salt&gt;
        </div>
        <div
          className="sc-EHOje jRprex"
          width={73.3952}
        >
          Sara is an energetic and curious person who has a keen interest in startups. Sara holds a bachelor's degree in business administration
    from Uppsala University, where, in addition to her studies, she was active in corporate promotion organizations. Since then, she has worked with
    marketing at Red Bull and most recently at the tech company Favro. When she is not working, she loves to train and is currently performing a Swedish
    classic.
        </div>
      </div>
    </div>
    <div
      className="sc-gzVnrw iyVpxV"
    >
      <div
        className="sc-bZQynM eIieJr"
      >
        <button
          className="sc-htoDjs gGWoSn"
          onClick={null}
        />
        <button
          className="sc-htoDjs aBrhj"
          onClick={[Function]}
        />
        <button
          className="sc-htoDjs aBrhj"
          onClick={[Function]}
        />
        <button
          className="sc-htoDjs aBrhj"
          onClick={[Function]}
        />
        <button
          className="sc-htoDjs aBrhj"
          onClick={[Function]}
        />
        <button
          className="sc-htoDjs aBrhj"
          onClick={[Function]}
        />
      </div>
    </div>
  </div>
</div>
`;
